ubyssey/static/src/styles/components/events/_event-submit-form.scss

```diff
@@ -33,6 +33,43 @@
   width: 100%;
 }
 
+.c-two-col {
+  // Structure
+  overflow: hidden;
+  width: 100%;
+}
+
+.c-two-col label {
+  display: block;
+}
+
+.c-two-col--right,
```

Comment:
Instead of left and right, you can just use a single `.c-form__row__col`

```scss
.c-form__row__col {
  flex: 1;

  padding-right: 0.75rem;
  padding-left: 0.75rem;
  
  &:first-child {
    padding-left: 0;
  }
  
  &:last-child {
    padding-right: 0;
  }
}